rush.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/rush.schema.json",
-  "rushVersion": "5.106.0",
+  "rushVersion": "5.99.0",
```

Comment:
can we add a comment in this file so this doesnt happen again?

Reply:
done